frontend/src/components/opportunity/OpportunityDocuments.tsx

```diff
@@ -0,0 +1,76 @@
+import { useTranslations } from "next-intl";
+import { Link, Table } from "@trussworks/react-uswds";
+
+interface OpportunityDocument {
+  opportunity_attachment_type: string;
+  file_name: string;
+  download_path: string;
+  updated_at: string;
+}
+
+interface OpportunityDocumentsProps {
+  documents: OpportunityDocument[];
+}
+
+const OpportunityDocuments = ({ documents }: OpportunityDocumentsProps) => {
+  const t = useTranslations("OpportunityListing.documents");
+
+  const documentTable = (documents_filtered: OpportunityDocument[]) => {
+    return (
+      <>
+        <thead>
+          <tr>
+            <th scope="col">{t("table_col_category")}</th>
+            <th scope="col">{t("table_col_file_name")}</th>
+            <th scope="col">{t("table_col_last_updated")}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {documents_filtered.map((document, index) => (
+            <tr key={index}>
+              <th data-label={t("table_col_category")} scope="row">
+                {document.opportunity_attachment_type ===
+                "notice_of_funding_opportunity"
+                  ? t("type.funding_details")
+                  : t("type.other")}
+              </th>
+              <td data-label={t("table_col_file_name")}>
+                <Link target="_blank" href={document.download_path}>
+                  {document.file_name}
+                </Link>
+              </td>
+              <td data-label={t("table_col_last_updated")}>
+                {new Date(document.updated_at).toDateString() +
```

Comment:
What I'm seeing in the UI doesn't match the designs as far as the timestamp format.
`Thu Oct 24 2024 16:05:21 GMT-0400 (Eastern Daylight Time)` vs `Oct 24, 2024 4:05:21 PM EDT`

We have a helper for this that should spit out the correct date string format for display https://github.com/HHS/simpler-grants-gov/blob/main/frontend/src/utils/dateUtil.ts. Using that should help, but may need to expand it in some way

Reply:
Yeah, I considered adding moment.js or something to get a very particular formatting, I guess my judgement call was it might be acceptable to deviate slightly from design r.e. exact formatting so long as all the info was there to avoid an external library and additional dependency, didn't know we had a dateUtil, I can look into that, but not sure if that will yield us the exact format in design either.

Reply:
Exploring libraries to help with this

Reply:
Group coalesced around day.js